### DISCUSSION TITLE
Multiple extruders temperature

### DISCUSSION THREAD

**Discussion by bokodimate** [UnchartedBull/OctoDash#3070] [Q&A] [answered]
Hi!

We have a leapfrog bolt pro. It has 2 extruders. 
My problem is that OctoDash only shows the T0 extruder's temperature even if I use the T1 extruder. 

Is there any way I can set OctoDash to show the active extruder's temperature? Or maybe to show both at the same time?

Thanks!

**Comment by smeisner** [comment]
As I have a FlashForge Creatore Pro with 2 extruders, I too would love to see support for multiple extruders.

Someone is already working on adding this to OctoDash (see https://github.com/UnchartedBull/OctoDash/issues/779)

**Comment by queengooborg** [comment, marked as answer]
Support has been added and will be available in v2.4.0!
